agent/apps/ai_guardrails/docs/api_spec.yaml

```
@@ -0,0 +1,110 @@
+                  enum: [input, output, both]
+                context:
+                  type: object
+      responses:
+        '200':
+          description: Evaluation result
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  allowed: { type: boolean }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)